xExtension-TitleWrap/static/title_wrap.css

```diff
@@ -0,0 +1,23 @@
+.horizontal-list {
+ display: flex;
+ /* uncomment to allow title to expand onto its own line */
+ /*flex-wrap: wrap;*/
```

Comment:
It's kind of useless to have this information here and it will be erased during an update. Since it can be set with the CustomCSS extension, you can give this CSS rule as an example in the README.